translations.tsv

```diff
@@ -1,238 +1,238 @@
+hQNoSeeds	Why can't it find my seed?	¿Por qué no encuentra mi semilla?	Pourquoi l'outil ne trouve t-il pas ma graine?
+hANoSeeds	"One mistake in the data you enter will be enough to stop MeteoNook from finding the correct seed. This tends to happen with weather types, as it's easy to mix them up. If you're not completely sure, then select the appropriate ""not sure"" option. It doesn't narrow down the options as quickly as selecting a specific type, but it's less risky - a slow but correct result is better than no result!"	"Un error en los datos que introduces es suficiente para que MeteoNook no pueda encontrar la semilla correcta. Esto suele pasar con los tipos del tiempo, porque es fácil confundir uno con otro. Si no estás completamente seguro, elige la opción ""No estoy seguro"" en la sección correspondiente. No elimina tantas opciones como si eligieras un tipo especifico, pero tiene menos riesgo - ¡un resultado correcto y lento es mejor que ningún resultado!"	"Une erreur dans les données fournies suffit à empecher MétéoNook de trouver la bonne graine. Ceci a tendance à arriver avec les types de conditions météo, vu qu'il est facile de les confondres. Si vous n'êtes pas totalement certain, choisissez une des options ""incertain"" qui s'y rapproche le plus. Cela ne reduira pas autant les combinaisons possibles que si vous choissisiez le type exact, mais c'est moins risqué - Obtenir lentement un resultat correct est meilleur que de ne pas en obtenir du tout !"
+hQBadSeed	It gave me a seed, but the results are wrong!	Me ha dado una semilla, ¡pero el pronóstico está mal!	Ca m'a donné une graine, mais les résultats sont faux !
+hABadSeed	"This happens sometimes, it sucks. What's most likely happened is that there was a mistake somewhere in your data and MeteoNook has found a seed which fits what you asked for, but doesn't actually match what your island uses. Try changing your weather types to ""not sure"", just in case it's a mistake with cloud types. If that doesn't help, you may have to collect new data."	"Esto pasa de vez en cuando. Lo más probable es que haya un error en tus datos y MeteoNook haya encontrado una semilla que encaja con tus datos, pero en verdad no es la semilla que tu isla tiene. Intenta cambiar los tipos del tiempo a ""no estoy seguro"", por si acaso es un error en los tipos de nubes. Si no ayuda, quizás tengas que obtener nuevos datos."	"Malheureusement, cela peut arriver. Ce qui s'est probablement passé est qu'il y a une erreur quelque part dans les données fournies et MétéoNook a trouvé une graine qui leurs correspond, meme si ce n'est pas celle de votre île. Essayez de changer ces données que vous avez fournis pour des types ""incertain"", au cas où ce soit une erreur dans les types de nuages ou autre. Si cela ne vous aide pas, essayer de collecter de nouvelles données.

```

Comment:
```suggestion
hABadSeed	This happens sometimes, it sucks. What's most likely happened is that there was a mistake somewhere in your data and MeteoNook has found a seed which fits what you asked for, but doesn't actually match what your island uses. Try changing your weather types to "not sure", just in case it's a mistake with cloud types. If that doesn't help, you may have to collect new data.	Esto pasa de vez en cuando. Lo más probable es que haya un error en tus datos y MeteoNook haya encontrado una semilla que encaja con tus datos, pero en verdad no es la semilla que tu isla tiene. Intenta cambiar los tipos del tiempo a "no estoy seguro", por si acaso es un error en los tipos de nubes. Si no ayuda, quizás tengas que obtener nuevos datos.	Malheureusement, cela peut arriver. Ce qui s'est probablement passé est qu'il y a une erreur quelque part dans les données fournies et MétéoNook a trouvé une graine qui leurs correspond, meme si ce n'est pas celle de votre île. Essayez de changer ces données que vous avez fournis pour des types "incertain", au cas où ce soit une erreur dans les types de nuages ou autre. Si cela ne vous aide pas, essayer de collecter de nouvelles données.
```